transpiler/src/main/java/org/jsweet/transpiler/Java2TypeScriptTranslator.java

```diff
@@ -5877,12 +5865,13 @@ public void visitAnnotation(JCAnnotation annotation) {
 				JCTree[] globalDecoratorFunction = context
 						.lookupGlobalMethod(((JCIdent) annotation.getAnnotationType()).sym.toString());
 				if (globalDecoratorFunction != null) {
-					if(!((JCMethodDecl)globalDecoratorFunction[1]).sym.getParameters().isEmpty()) {
+					if (!((JCMethodDecl) globalDecoratorFunction[1]).sym.getParameters().isEmpty()) {
 						parens = false;
 					}
 				}
 			}
-			if(parens) print("()");
+			if (parens)
```

Comment:
Même si c'est un détail : peut on avoir les accolades pour l'homogénéité ? Et éviter la fameuse erreur d'inattention